sphinxgallery/gen_rst.py

```diff
@@ -445,7 +445,7 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf, plot_gallery):
     Returns the set of functions/classes imported in the example.
     """
     base_image_name = os.path.splitext(fname)[0]
-    image_fname = '%s_%%03d.png' % base_image_name
+    image_fname = 'sphx_glr_%s_%%03d.png' % base_image_name
```

Comment:
Why the sphx_glr_ prefix ? I don't think it adds anything since the image name already contains the name of the example script.

Also this is going to break the nilearn doc because we manually includes generated images (do `git grep -P 'plot.*.png' doc` to get all of them). I am guessing this is the same for scikit-learn.